test-cypress/steps/sparql-steps.js

```diff
@@ -0,0 +1,119 @@
+class SparqlSteps {
+
+    static changeLanguage(language) {
+        // Change the language
+        cy.get('#languageGroupDrop').should('be.visible').click().then(() => {
```

Comment:
It's OK this way, but generally, you don't need to chain the next command in the promise callback. It should also work like:
```
cy.get('#languageGroupDrop').should('be.visible').click();
cy.get('.dropdown-menu .dropdown-item').contains(language).click();
```

Reply:
OK, I'll keep this in mind.